### DISCUSSION TITLE
Not same images with same prompt, seed, etc, and not as sharp as before upgrade

### DISCUSSION THREAD

**Discussion by Chinico68** [vladmandic/sdnext#1641] [Q&A]
⚠️What happened to SD Vlad? Images are not as sharp as they used to be, and also using same seed, prompt, steps etc. from a previous made image I still don't get the same image how is that possible? I can't replicate any images from my last update (yesterday), I moved all the checkpoints and safetensors to an external drive cause  main disk was full, but I don't think that could be an issue!? Or could it be? 😡😡😡😡😡😡⚠️
Apple studio M1 Max - OS Ventura 13.4.1 (22F82) - Torch 2.0.0 - Python 3.10.6

**Comment by Tillerz** [comment]
Can NOT confirm. I had `Version: 768daecd Mon Jul 10 08:36:46 2023 -0400` right now, rendered an image, updated to `Version: a844a83d Tue Jul 11 11:00:45 2023 +0300`, rendered again, with same seed, output is the same.
